e2e/iam/org_invitations_test.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e iam
+
+package iam_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	roleNameOrg = "ORG_OWNER"
+	emailOrg    = "test1@mongodb.com"
+)
+
+func TestOrgInvitations(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	var OrgInvitationID string
+
+	t.Run("Invite", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			orgEntity,
+			invitationsEntity,
+			"invite",
+			emailOrg,
+			"--role",
+			"ORG_MEMBER",
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		a := assert.New(t)
+		a.NoError(err, string(resp))
```

Comment:
[nit] let's change this to `require.NoError`, if it fails you can't move forward